compiler/x/codegen/OMRInstruction.hpp

```diff
@@ -198,7 +198,9 @@ class OMR_EXTENSIBLE Instruction : public OMR::Instruction
    virtual bool dependencyRefsRegister(TR::Register *reg);
 
    virtual uint8_t *generateBinaryEncoding();
-   virtual int32_t estimateBinaryLength(int32_t currentEstimate);
+   virtual int32_t  estimateBinaryLength(int32_t currentEstimate);
+   virtual uint8_t* operand(uint8_t* cursor) { return cursor; }
```

Comment:
What are the semantics of this function? May I suggest to include some Doxygen documentation as it is not intuitive what purpose this function serves.